anisoap/representations/radial_basis.py

```diff
@@ -1,4 +1,87 @@
+import warnings
+
 import numpy as np
+from equistore.core import TensorMap
+from scipy.special import gamma
+
+
+def inverse_matrix_sqrt(matrix: np.array):
+    """
+    Returns the inverse matrix square root.
+    The inverse square root of the overlap matrix (or slices of the overlap matrix) yields the
+    orthonormalization matrix
+    Args:
+        matrix: np.array
+            Symmetric square matrix to find the inverse square root of
+
+    Returns:
+        inverse_sqrt_matrix: S^{-1/2}
+
+    """
+    if not np.allclose(matrix, matrix.T):
+        raise ValueError("Matrix is not hermitian")
+    eva, eve = np.linalg.eigh(matrix)
+
+    if (eva < 0).all():
+        raise ValueError(
+            "Matrix is not positive semidefinite. Check that a valid gram matrix is passed."
+        )
+    return eve @ np.diag(1 / np.sqrt(eva)) @ eve.T
```

Comment:
Any reason for not using cholesky? 

https://stackoverflow.com/questions/71232192/efficient-matrix-square-root-of-large-symmetric-positive-semidefinite-matrix-in

Reply:
I don't think cholesky decomp == square root, since cholesky decomp is $A=LL^T$, not $A=LL$. 

This is how inverse square root was implemented by librascal and rascaline so I just used their implementation, and . If we are getting hung up on speed on this step in the future we can possibly explore a faster option.